libraries/standard/mqtt/cbmc/proofs/MQTT_GetSubAckStatusCodes/MQTT_GetSubAckStatusCodes_harness.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
```

Comment:
Can you please add the same copyright as the other harnesses

Reply:
Added